docs/mutation-testing.md

```diff
@@ -0,0 +1,109 @@
+# Mutation Testing
+
+This document describes how to run mutation testing locally to mimic the outcome of a PR, without the CI limitation it provides by timing out after 6 hours.
+[Here is the script](../contrib/tools/local-mutation-testing.sh) to run mutation locally running the mutants created by the changes between the current branch and develop. It does automatically all the steps explained below.
+
+From the root level of the stacks-core repository run
+```sh
+./contrib/tools/local-mutation-testing.sh
+```
+
+## Prerequirements
+
+Install the cargo mutants library
+```sh
+cargo install --version 24.7.1 cargo-mutants --locked
+```
+
+
+## Steps 
+1. be on source branch you would use for the PR.
```

Comment:
double-check the capitalization here - same nit for each numbered list that follows.